### DISCUSSION TITLE
How to customize font weight?

### DISCUSSION THREAD

**Discussion by neulad** [rainbow-me/rainbowkit#933] [Q&A]
I successfully changed the font of the `ConnectButton`. However I ddidn't find the reference to change the font weight of the font. How to do that? Thanks!

**Comment by ghost** [comment]
I'm interested on that too. Any suggestions?

**Comment by ghost** [comment]
> I successfully changed the font of the `ConnectButton`. However I ddidn't find the reference to change the font weight of the font. How to do that? Thanks!

I found the solution, you can do it by adjusting initial library code. Inside of ConnectButton() function you will find related fontWeight property. My guess it can take only predefined values listed below: 

fontWeight?: "regular" | "medium" | "semibold" | "bold" | "heavy" | undefined;

**Reply by neulad** [reply]
What  version of Rainbow are you using?

**Reply by ghost** [reply]
"@rainbow-me/rainbowkit": "^0.9.0"
Previously, it was one of the latest versions. But it was conflicting with rainbowkit-siwe module, so it somehow was changed to that version (perhaps through pack install).
